WordPress/Classes/ViewRelated/Blog/Blog Details/Detail Header/BlogDetailHeaderView.swift

```diff
@@ -151,37 +141,17 @@ class BlogDetailHeaderView: UIView {
     private var topActionRowConstraint: NSLayoutConstraint?
 
     private func setupConstraintsForChildViews(_ showsActionRow: Bool) {
-        var constraints = constraintsForTitleView()
-
-        if !FeatureFlag.mySiteDashboard.enabled {
-            constraints += constraintsForActionRow(showsActionRow)
-        }
+        let constraints = constraintsForTitleView()
 
         NSLayoutConstraint.activate(constraints)
     }
 
-    private func constraintsForActionRow(_ showsActionRow: Bool) -> [NSLayoutConstraint] {
-        let widthConstraint = actionRow.widthAnchor.constraint(equalToConstant: LayoutSpacing.maxButtonWidth)
-        widthConstraint.priority = .defaultHigh
-
-        let topActionRowConstraint = actionRow.topAnchor.constraint(equalTo: titleView.bottomAnchor, constant: LayoutSpacing.betweenTitleViewAndActionRow(showsActionRow))
-        self.topActionRowConstraint = topActionRowConstraint
-
-        return [
-            topActionRowConstraint,
-            actionRow.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -LayoutSpacing.belowActionRow),
-            actionRow.leadingAnchor.constraint(greaterThanOrEqualTo: titleView.leadingAnchor),
-            actionRow.trailingAnchor.constraint(lessThanOrEqualTo: titleView.trailingAnchor),
-            actionRow.centerXAnchor.constraint(equalTo: centerXAnchor),
-            widthConstraint
-        ]
-    }
-
     private func constraintsForTitleView() -> [NSLayoutConstraint] {
         var constraints = [
             titleView.topAnchor.constraint(equalTo: topAnchor, constant: LayoutSpacing.top),
             titleView.leadingAnchor.constraint(equalTo: safeAreaLayoutGuide.leadingAnchor, constant: LayoutSpacing.atSides),
-            titleView.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor, constant: -LayoutSpacing.atSides)
+            titleView.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor, constant: -LayoutSpacing.atSides),
+            titleView.bottomAnchor.constraint(equalTo: bottomAnchor)
         ]
 
         if FeatureFlag.mySiteDashboard.enabled {
```

Comment:
We might not need this entire block anymore, since we're always constraining to the bottom anchor.

Reply:
Just to clarify, I mean the `if FeatureFlag.mySiteDashboard.enabled {...}` block.

Reply:
Fixed in 78aaebe5824694d3567bd05a528f6b71df74560f